meetings/cranelift/2022/cranelift-01-10.md

```diff
@@ -0,0 +1,21 @@
+# January 10 project call
+
+**See the [instructions](../README.md) for details on how to attend**
+
+## Agenda
+1. Opening, welcome and roll call
+    1. Note: meeting notes linked in the invite.
+    1. Please help add your name to the meeting notes.
+    1. Please help take notes.
+    1. Thanks!
+1. Announcements
+    1. _Sumbit a PR to add your announcement here_
```

Comment:
Don't know if this was a typo that became a recurring joke, but was always amused by this `Sumbit` here and below ;)

Reply:
Many eyeballs have passed over this but yours are the first to see the typo! Thanks!